### ISSUE TITLE
GCP Workload Identity support with Connector

### ISSUE THREAD

**Issue by singh-ashutosh1** [confluentinc/kafka-connect-bigquery#232]
BigQuery Connector does not support GCP workload identity.

**Comment by anton-hi** [comment]
I agree this would be very useful. I believe the requirement is to be able to authenticate to BigQuery using workload identity for scenarios where the Connect worker nodes are hosted on GCP, including Confluent Platform running on GCE (Compute Engine) and CFK running on GKE (Kubernetes Engine).

**Comment by atrbgithub** [comment]
@singh-ashutosh1 @anton-hi What happens if you try this? Is an error thrown?

The docs [here](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#using_from_your_code) say that:

> Authenticating to Google Cloud services from your code is the same process as authenticating using the [Compute Engine metadata server](https://cloud.google.com/compute/docs/storing-retrieving-metadata).

I assume kafka-connect doesn't attempt to use the metadata service and just throws a permission denied error....?

**Comment by rashmi1s** [comment]
Hi,

Do we have any updates on workload identity support with Bigquery kafka connector ?
